interfaces/builtin/framebuffer.go

```diff
@@ -83,17 +82,6 @@ func (iface *FramebufferInterface) AppArmorConnectedPlug(spec *apparmor.Specific
 
 // Getter for the security snippet specific to the plug
 func (iface *FramebufferInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
-	switch securitySystem {
```

Comment:
Looks better now=)

Does it mean that udev tagging will never be enabled for the framebuffer interface?